CHANGELOG.rst

```diff
@@ -4,6 +4,23 @@ Ansible Playbooks for Confluent Platform - Release Notes
 
 .. contents:: Topics
 
+v7.6.0
+======
+
+New features
+-------------
+
+- In place upgrade from Zookeeper to Kraft is now supported
+- ARM64 architecture is now supported
+
+Notable enhancements
+-------------
+
+- Oracle8, Oracle9, Rocky8, Rock9 Operating Systems are now supported
+- Discovery is now a separate repo named cp-discovery
+- Connectors now supported on clusters with RBAC & mTLS
+- Support for custom port for kraft controller added
```

Comment:
this should be a little more explanatory, something like "Custom port can now be used in Kraft Controller listener"